code/controllers/subsystem/input.dm

```diff
@@ -95,7 +95,7 @@ SUBSYSTEM_DEF(input)
 // For initially setting up or resetting to default the movement keys
 /datum/controller/subsystem/input/proc/setup_default_movement_keys()
 	var/static/list/default_movement_keys = list(
-		"W" = NORTH, "A" = WEST, "S" = SOUTH, "D" = EAST,				// WASD
+//		"W" = NORTH, "A" = WEST, "S" = SOUTH, "D" = EAST,				// WASD are now keybinds, not hard-coded.
```

Comment:
The only thing I'm unsure about is if this should stay commented out, or just be removed.

Reply:
That's fair. I've removed it.